test/verify/check-machines

```diff
@@ -88,6 +160,9 @@ class TestMachines(MachineCase):
         b.click("#vm-subVmTest1-forceOff")
         b.wait_in_text("#vm-subVmTest1-state", "shut off")
 
+        output = self.machine.execute("cat /var/log/libvirt/qemu/console-subVmTest1.log")
+        self.assertIn("Linux version", output)
```

Comment:
This is a race: the VM might not have booted enough to put "Linux version" in the log before it gets shut down.
We can put this before the shutdown:

```
wait(lambda: "Linux version" in self.machine.execute("cat /var/log/libvirt/qemu/console-subVmTest1.log"))
```

  (This actually happened locally here, reliably.)